src/unix_permissions.rs

```diff
@@ -0,0 +1,9 @@
+/// A NOOP struct for bringing the API between Windows and Unix up to parity. To set permissions
+/// properly on Unix, you can just use `std::os::unix::fs::PermissionsExt`.
+pub struct SecurityAttributes;
+
+impl SecurityAttributes {
+    pub fn empty() -> Self { SecurityAttributes }
```

Comment:
Documentation is lacking here. I also don't like the fact that we have a different return type for this and windows version - this will make writing generic code more difficult. Will create an issue for this.

Reply:
Unless you're addressing the fact that this is a no-op on Unix, then I don't see the issue. As exported from the crate, there is no practical difference between `win_permissions::SecurityAttributes` and `inux_permsissions::SecurityAttributes`, as currently the branch for integrating these changes into `jsonrpc-ipc-server` does not contain any extra `#[cfg($platform)]`.

Reply:
If I call `SecurityAttributes::allow_everyone_connect()` in the paltform-generic code it will compile only on one of the platforms, since the unix version returns just `Self`, while windows returns `io::Result`.

`empty()` is actually fine, I misplaced the comment.

Reply:
Disregard that, I'm just plain wrong here.

Reply:
#8